src/main/scala/Garage/DBConnection.scala

```diff
@@ -0,0 +1,15 @@
+package Garage
+
+import org.mongodb.scala._
+object DBConnection extends App {
+  val mongoClient: MongoClient = MongoClient("mongodb://localhost")
+
+  def getConnection():Unit = {
```

Comment:
this should be two methods, one getting the database, another getting the collection. additionally you shouldn't hard code them but take them as arguments